docs/DeveloperGuide.md

```diff
@@ -727,11 +727,19 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **Extensions**
 - 2a. The user does not have any free slots available.
+<<<<<<< HEAD
+  - 2a1. System informs user that there is no available timeslots.
+    <br>
+    <br>
+  
+**Use case: UC12 - Calculating monthly revenue**
+=======
```

Comment:
Should we remove the merge conflict arrows so that it is not reflected in the DG?